rules/linux/process_creation/proc_creation_lnx_pnscan_binary_cli_pattern.yml

```
@@ -0,0 +1,22 @@
+title: Detection of Pnscan Binary Data Transmission Behavior
+id: 97de11cd-4b67-4abf-9a8b-1020e670aa9e
+status: experimental
+description: Detects command line patterns associated with the use of pnscan for sending binary data across the network. This behavior has been identified in a Linux malware campaign targeting Docker, Apache Hadoop, Redis, and Confluence.
+author: David Burkett (@signalblur)
+date: 2024-04-03
+references:
+    - https://www.cadosecurity.com/blog/spinning-yarn-a-new-linux-malware-campaign-targets-docker-apache-hadoop-redis-and-confluence
+    - https://regex101.com/r/RugQYK/1
+logsource:
+    category: process_creation
+    product: linux
+detection:
+    selection:
+        CommandLine|re: '.*(-(W|R)( |")(([0-9a-fA-F]{2}) ){2,20}).*'
+    condition: selection
+falsepositives:
+    - Legitimate use of pnscan for network diagnostics or authorized scanning activities.
+level: high
+tags:
+    - linux
+    - pnscan
```

HI,
https://github.com/SigmaHQ/sigma-specification/blob/main/Tags_specification.md
```suggestion
tags:
    - attack.discovery
    - attack.t1046
```